crates/sui-json-rpc-api/src/read.rs

```diff
@@ -59,6 +59,20 @@ pub trait ReadApi {
         options: Option<SuiObjectDataOptions>,
     ) -> RpcResult<Vec<SuiObjectResponse>>;
 
+    /// Given a list of object IDs, return the information of each object needed for transaction building.
+    /// If any object does not exist, it will return an error.
+    /// If any object is an object-owned object, which cannot be used as transaction input, it will return an error.
```

Comment:
I'm assuming this is referring to child objects (e.g. dynamic fields) rather than objects that have been transferred to other objects -- this may be worth clarifying, because I think for most builders "object-owned object" suggests the transfer-to-object case (which can be used as a transaction input, it just needs to be a `Receiving` input), and not the dynamic field case.